package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-find-rules",
-  "version": "3.1.0",
+  "version": "3.1.1",
```

Comment:
generally versions should never be bumped in PRs, only in commits directly to master (which i assume you were doing for the "bump" PRs) - is that what you're planning to do here?